atom-publisher-lib/build.sbt

```diff
@@ -18,7 +18,14 @@ startDynamoDBLocal <<= startDynamoDBLocal.dependsOn(compile in Test)
 test in Test <<= (test in Test).dependsOn(startDynamoDBLocal)
 testOptions in Test <+= dynamoDBLocalTestCleanup
 
+dependencyOverrides += "org.apache.thrift" % "libthrift" % "0.9.1"
```

Comment:
This is preventing atom-maker from picking up newer versions of the thrift/scrooge dependencies (from fezziwig). We've caused issues for capi clients in the past when we've bumped the scrooge version, so this is to ensure nothing changes.